### ISSUE TITLE
The cipher modes of operation should be enum constants

### ISSUE THREAD

**Issue by daviditen** [chapel-lang/chapel#9491]
The cipher modes of operation currently used by AES and Blowfish should be constants in an enum in the Crypto module.  Where a string is needed for them, casting the constant to string type should get the right string.

Explicitly enumerating the possible modes would be a nice little UI improvement.

Something like this should do it:

```
enum CryptoChainMode {
  cbc, ecb, cfb, ofb /*, others? */
}


proc needsStringArg(x: string) {
  writeln(x);
}

needsStringArg(CryptoChainMode.ecb: string);
```

**Comment by daviditen** [comment]
@saru95 Any interest in taking on this UI improvement?

**Comment by saru95** [comment]
@daviditme I'll take this up.

**Comment by daviditen** [comment]
Closed with PR #9556.
